io.catenax.mandatory_dismantling/2.0.0/MandatoryDismantling.ttl

```diff
@@ -0,0 +1,944 @@
+#######################################################################
+# Copyright (c) 2023,2024 ISTOS GmbH (a member of the DMG Mori Group)
+# Copyright (c) 2023,2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
+# Copyright (c) 2023,2024 TRUMPF Werkzeugmaschinen SE + Co. KG
+# Copyright (c) 2023,2024 Volkswagen AG
+# Copyright (c) 2023,2024 BASF SE
+# Copyright (c) 2023,2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2023,2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IML)
+# Copyright (c) 2023,2024 Henkel AG & Co. KGaA
+# Copyright (c) 2023,2024 Mercedes Benz AG
+# Copyright (c) 2023,2024 SAP SE
+# Copyright (c) 2023,2024 SupplyOn AG
+# Copyright (c) 2023,2024 ZF Friedrichshafen AG
+# Copyright (c) 2023,2024 CGI Deutschland B.V. & Co. KG
+# Copyright (c) 2023,2024 T-Systems International GmbH
+# Copyright (c) 2023,2024 Contributors to the Eclipse Foundation
```

Comment:
```suggestion
# Copyright (c) 2023 ISTOS GmbH (a member of the DMG Mori Group)
# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
# Copyright (c) 2023 TRUMPF Werkzeugmaschinen SE + Co. KG
# Copyright (c) 2023 Volkswagen AG
# Copyright (c) 2023 BASF SE
# Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IML)
# Copyright (c) 2023 Henkel AG & Co. KGaA
# Copyright (c) 2023 Mercedes Benz AG
# Copyright (c) 2023 SAP SE
# Copyright (c) 2023 SupplyOn AG
# Copyright (c) 2023 ZF Friedrichshafen AG
# Copyright (c) 2023 CGI Deutschland B.V. & Co. KG
# Copyright (c) 2023 T-Systems International GmbH
# Copyright (c) 2023 Contributors to the Eclipse Foundation
```

I also updated the licence-header, as the TRG has been updated slightly. Only the year of the first contribution is maintained as everything else can be derived from the Github commit log.